### PR TITLE
Fix motor stop

### PR DESCRIPTION
**Pull request by haslinghuis**
- [x] Disabling motor testing and returning to the motors tab will re-enable testing.
- [x] Resetting status polling to run all the time instead of just during engine testing fixes this bug.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by Asizon — APPROVED** [review]

**Review by chmelevskij — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
